personal/privacyidea/class_mfaAccount.inc

```diff
@@ -192,8 +192,8 @@ class mfaAccount extends plugin
         /* Call parent execute */
         plugin::execute();
 
-        if (!$this->is_account && $this->acl_is_writeable("")) {
-            $this->is_account = true;
```

Comment:
This should stay, it means: if an LDAP object does not have mfaService as objectClass and the editing user as permission to write the whole object (maybe better: create? We need to check other locations in GOsa² if something similar exists there) the UI shall pretend as if mfaService is already set. When saving/applying the plugin page content, the mfaService objectClass + attributes will be set.